integration-tests/c-example/CMakeLists.txt

```diff
@@ -11,4 +11,6 @@ set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 set(CMAKE_EXPORT_LINK_COMMANDS ON)
 
 add_subdirectory(lib)
-add_subdirectory(src)
\ No newline at end of file
+add_subdirectory(src)
+add_subdirectory(lib1)
```

Comment:
c-example project that we can show for other people, can we avoid complication of it's structure?

Reply:
Done